.github/workflows/master.yml

```diff
@@ -91,7 +91,6 @@ jobs:
         # We cache R dependencies based on a tuple of the current OS, the R version, and the list of
         # R dependencies
         key: ${{ steps.os-name.outputs.os-name }}-${{ hashFiles('.github/R-version') }}-1-${{ hashFiles('.github/depends.Rds') }}
-        restore-keys: ${{ steps.os-name.outputs.os-name }}-${{ hashFiles('.github/R-version') }}-1-
```

Comment:
Remove restore-keys to enforce invalidating cache when `.github/depends.Rds` which depends on the `DESCRIPTION` file.

Reply:
This should prevent restoring cache that contains packages that the DESCRIPTION file doesn't.